tracer/test/Datadog.Trace.Tests/Util/Http/QueryStringManagerTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿// <copyright file="QueryStringManagerTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.Logging;
+using Datadog.Trace.Util.Http;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace Datadog.Trace.Tests.Util.Http
+{
+    public class QueryStringManagerTests
+    {
+        [Theory]
+        [InlineData(false, 5000, null, "")]
+        [InlineData(true, 2, null, "ab")]
+        [InlineData(true, 0, null, "abcde")]
+        [InlineData(true, 0, ".*", "<redacted><redacted>")]
```

Comment:
nit: I would also move `inputstring` to be an input variable and test `null` and `""` to get the edge cases

Reply:
Thanks. Fixed in 72e7e28d04bd04864b78611fd37a823d18008eba